src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -614,6 +615,64 @@ internal static void RegisterMemoryLoadChangeNotification(float lowMemoryPercent
             }
         }
 
+        [Flags]
+        internal enum FinalizerActionKind
```

Comment:
NIT: Probably worth adding that the corresponding enum on the VM side of things is in gcinterface.h to make sure readers note both instances if changes are needed to be made.

Reply:
It is already defined in `gcinterface.h` with the comments.